aws/bastion-userdata.tmpl

```diff
@@ -24,7 +24,6 @@ export DEBIAN_FRONTEND=noninteractive
 info Updating packages. . .
 apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" update
 apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" upgrade
-apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" dist-upgrade
```

Comment:
May I ask why?

Reply:
TL;DR a dist-upgrade was interrupting the user-data / startup-script in some cases because it was increasing the number of times systemd was being reloaded. Internal discussion yielded it is less necessary to dist-upgrade as nightly updates also happen, plus we should use newest images.

Ref: https://fairwindsops.slack.com/archives/C0198T60J23/p1613062235012400